src/EVM/SymExec.hs

```diff
@@ -62,6 +60,16 @@ isQed :: ProofResult a b c -> Bool
 isQed (Qed _) = True
 isQed _ = False
 
+sameCnstr :: ProofResult a b c -> ProofResult a b c -> Bool
```

Comment:
I think we can clean this up like this? 

https://stackoverflow.com/a/10112843